spec/requests/sessions_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe "Sessions", type: :request do
+
+  let(:user1) { create(:user) } 
+
+  describe "#create" do
+
+    context "user not found" do
+
+      it "with invalid email" do
```

Comment:
``it`` message should display test output. For example ``it "redirects to login form"``.